jtravis/src/test/java/fr/inria/spirals/jtravis/helpers/BuildHelperTest.java

```diff
@@ -346,4 +349,37 @@ public void testGetFutureBuildOfBranch2() {
         assertEquals("21420", obtained.getNumber());
     }
 
+    @Test
+    public void testGetBuildsFromSlugWithLimitDate() {
+        String slug = "surli/failingProject";
+        Date limitDate = TestUtils.getDate(2017, 9, 14, 0, 0, 1);
+
+        List<Build> builds = BuildHelper.getBuildsFromSlugWithLimitDate(slug, limitDate);
```

Comment:
So you're getting all builds until this date, right? You should specify it in a comment.